src/index.js

```diff
@@ -10,7 +10,7 @@ export default function wrap(classes, prefix) {
   prefix = prefix || " "
 
   if (Array.isArray(classes) && classes.length) {
-    for (var i = 0; i < classes.length; i++) {
+    for (var i = 0, l = classes.length; i < l; i++) {
```

Comment:
@drdla What is this magic? 😮😄

Reply:
Ah, you are one time caching the length of the array, okay. Like you said I doubt we'll see a big improvement, but this LGTM!